docs/other-debugging-methods.md

```diff
@@ -0,0 +1,97 @@
+---
+id: other-debugging-methods
+title: Other Debugging Methods
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+This page covers other JavaScript debugging methods besides what is described in [Opening the Debugger](./debugging#opening-the-debugger).
+
+## Safari Developer Tools (direct JSC debugging)
```

Comment:
Elsewhere in the docs, are we calling it JSC or JavaScriptCore? Do we have a JSC page we could link to for better context?